.github/workflows/action-test-frontend.yml

```diff
@@ -0,0 +1,42 @@
+name: Frontend Tests
+
+on:
```

Comment:
Sorry for not putting it as a requirement in the Jira, but we would ideally want the tests to be run only on PRs to develop, master.

Reply:
it is now changed to run only on PR on develop and master